### PR TITLE
#567 Ease assertions on content of JSON files

### PR DESCRIPTION
**Pull request by fstaudt**

### CODE REVIEW

File: json-unit-assertj/src/main/java/net/javacrumbs/jsonunit/assertj/JsonFileAssert.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+     * Assert json properties in a file from its path.
+     *
+     * <pre>{@code
+     * assertThatJsonFile("path/to/file.json").isFile()
+     *     .hasContent().inPath("test1").isEqualTo(2);
+     * }</pre>
+     */
+    @NotNull
+    public static JsonFileAssert assertThatJsonFile(@NotNull String path) {
+        return new JsonFileAssert(new File(path));
```
[the remark this answers is not in the document]
Comment (fstaudt) [reply json-unit-assertj/src/main/java/net/javacrumbs/jsonunit/assertj/JsonFileAssert.java:33]: @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (fstaudt) [reply json-unit-assertj/src/main/java/net/javacrumbs/jsonunit/assertj/JsonFileAssert.java:33]: Vulnerability identified by lift is only applicable for API.

Not relevant in the case of an assertion for unit tests

**Comment by lukas-krecan** [comment]
Hi, thank you. I do not think this belongs to the library as everyone can implement a method like this. I am trying (and failing) to keep the library as small as possible.

**Comment by fstaudt** [comment]
> Hi, thank you. I do not think this belongs to the library as everyone can implement a method like this. I am trying (and failing) to keep the library as small as possible.

I've indeed implemented the assertion class first in [my project](https://github.com/fstaudt/helm-values) and I had to duplicate it in all my modules.

I thought it would have added value to move the class to the library, it's up to you to decide.

Thanks already for the library, it already eased my tests on JSON files.

**Pull request closed without merging**